doc/nrf/releases_and_maturity/releases/release-notes-3.0.0.rst

```diff
@@ -0,0 +1,1349 @@
+
+  * The Advertising Coding Selection feature to the following samples:
+
+    * :ref:`bluetooth_central_hr_coded`
+    * :ref:`peripheral_hr_coded`
+
+* Updated:
+
+  * The configurations of the non-secure ``nrf5340dk/nrf5340/cpuapp/ns`` board target in the following samples to properly use the TF-M profile instead of the predefined minimal TF-M profile:
+
+    * :ref:`bluetooth_central_hids`
+    * :ref:`peripheral_hids_keyboard`
+    * :ref:`peripheral_hids_mouse`
+
+    This change results from the Bluetooth subsystem transition to the PSA cryptographic standard.
+    The Bluetooth stack can now use the PSA crypto API in the non-secure domain as all necessary TF-M partitions are configured properly.
+
+  * The configurations of the following samples by increasing the main thread stack size (:kconfig:option:`CONFIG_MAIN_STACK_SIZE`) to prevent stack overflows:
+
+    * :ref:`bluetooth_central_hids`
+    * :ref:`peripheral_hids_keyboard`
+    * :ref:`peripheral_hids_mouse`
+
+    This change results from the Bluetooth subsystem transition to the PSA cryptographic API.
+
+  * The following samples to use LE Secure Connection pairing (:kconfig:option:`CONFIG_BT_SMP_SC_PAIR_ONLY`).
+
+    * :ref:`peripheral_gatt_dm`
+    * :ref:`peripheral_mds`
+    * :ref:`peripheral_cts_client`
+
+* :ref:`direct_test_mode` sample:
+
+  * Added:
+
+    * Loading of radio trims and a fix of a hardware errata for the nRF54H20 SoC to improve the RF performance.
+    * Support for the :ref:`nRF2220 front-end module <ug_radio_fem_nrf2220ek>`.
+    * Workaround for the hardware errata HMPAN-216 for the nRF54H20 SoC.
+
+* :ref:`central_uart` sample:
+
+  * Added reconnection to bonded devices based on their address.
+
+* :ref:`power_profiling` sample:
+
+  * Added the :kconfig:option:`CONFIG_BT_POWER_PROFILING_NFC_DISABLED` Kconfig option to reduce the power consumption by disabling the NFC.
+
+Bluetooth Mesh samples
+----------------------
+
+* Added:
+
+  * Support for nRF54L10 in the following samples:
+
+    * :ref:`bluetooth_mesh_sensor_client`
+    * :ref:`bluetooth_mesh_sensor_server`
+    * :ref:`bluetooth_ble_peripheral_lbs_coex`
+    * :ref:`bt_mesh_chat`
+    * :ref:`bluetooth_mesh_light_switch`
+    * :ref:`bluetooth_mesh_silvair_enocean`
+    * :ref:`bluetooth_mesh_light_dim`
+    * :ref:`bluetooth_mesh_light`
+    * :ref:`ble_mesh_dfu_target`
+    * :ref:`bluetooth_mesh_light_lc`
+    * :ref:`ble_mesh_dfu_distributor`
+
+  * Support for nRF54L05 in the following samples:
+
+    * :ref:`bluetooth_mesh_sensor_client`
+    * :ref:`bluetooth_mesh_sensor_server`
+    * :ref:`bluetooth_ble_peripheral_lbs_coex`
+    * :ref:`bt_mesh_chat`
+    * :ref:`bluetooth_mesh_light_switch`
+    * :ref:`bluetooth_mesh_silvair_enocean`
+    * :ref:`bluetooth_mesh_light_dim`
+    * :ref:`bluetooth_mesh_light`
+    * :ref:`bluetooth_mesh_light_lc`
+
+* Updated the following samples to include the value of the :kconfig:option:`CONFIG_BT_COMPANY_ID` option in the Firmware ID:
```

Comment:
```suggestion
* Updated:

  * The board configuration files for targets nRF54L15, nRF54L10 and nRF54L05 by increasing (:kconfig:config:`CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE`) and (:kconfig:config:`CONFIG_BT_RX_STACK_SIZE`) to prevent stack overflows when (:kconfig:config:`CONFIG_BT_MESH_USES_MBEDTLS_PSA`) is selcted.

  * The following samples to include the value of the :kconfig:option:`CONFIG_BT_COMPANY_ID` option in the Firmware ID:

    * :ref:`ble_mesh_dfu_distributor`
    * :ref:`ble_mesh_dfu_target`

  * Disabling the friend feature when the following sample is compiled for the :ref:`zephyr:nrf52dk_nrf52832` board target to increase the amount of RAM available for the application.

    * :ref:`bluetooth_mesh_light_lc`
```